client/packages/programs/src/JsonForms/common/components/Text.tsx

```diff
@@ -136,7 +147,6 @@ const UIComponent = (props: ControlProps) => {
         FormHelperTextProps: error
           ? { sx: { color: 'error.main' } }
           : undefined,
-        required: props.required,
```

Comment:
I first added it here as well but then it shows a "required *" which we don't show anywhere else. Think just marking the text box as red is clear enough to indicate that the field is required.

Reply:
agree - best to be consistent. would be good to have a required indicator throughout, but we do need to discuss and co-ordinate before doing that!